lib/main.dart

```diff
@@ -22,15 +24,16 @@ class MyApp extends StatelessWidget {
 class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
+    /*return Scaffold(
       appBar: AppBar(
         title: Text("Lingua"),
       ),
       body: Padding(
         padding: EdgeInsets.all(16.0),
         child: Text("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."),
       )
-    );
+    );*/
```

Comment:
Why is the code commented here? Please no commented code on production.

Reply:
Oups! Just forgot to clean it